docs/topics/kotlin-ide.md

```diff
@@ -15,6 +15,22 @@ Kotlin plugin is bundled with each IntelliJ IDEA release.
 
 Read more about IntelliJ IDEA in the [official documentation](https://www.jetbrains.com/help/idea/discover-intellij-idea.html).
 
+## Fleet
+
+> JetBrains Fleet is currently in Public Preview, and at this stage is free to use.
```

Comment:
```suggestion
> JetBrains Fleet is currently in Public Preview, and at this stage is free to use.
>
```

Reply:
There should be an extra line with the ">" sign before {type=""}.
Otherwise, the note will be fallback to the regular gray "tip"